src/app/cmdoptions.cpp

```diff
@@ -516,53 +516,55 @@ QString makeUsage(const QString &prgName)
     QString indentation = QString(USAGE_INDENTATION, ' ');
 
     stream << QObject::tr("Usage:") << '\n';
-    stream << indentation << prgName << QLatin1String(" [options] [(<filename> | <url>)...]") << '\n';
+    stream << indentation << prgName << QLatin1String(" [OPTIONS] [URI...]") << '\n';
```

Comment:
note on coding style, I would prefer concatenate the stream together:
```c++
steam << //.. no need to end this expression with ';'
    << //.. more
    << /* end */ ;
```

Reply:
@Chocobo1
I applied the suggested change, with the caveat that I added an extra indentation level to `<< wrapText(...`s that are the description of the option name above them. This way it is easy to see where the lines corresponding to one option start and and end. Let me know if this is undesirable in your opinion.
In addition:
- all lines have now length < 100
- there is an empty line separating logical groups of options (as before)
- The first `<< '\n'` is always in the same line as other text. If there is a second, it is in the next line (same as before).

Reply:
Thanks! looks good to me.